### PR TITLE
Update Rule “do-you-re-purpose-your-pillar-content-for-social-media/rule”

### PR DESCRIPTION
**Pull request by camillars**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by tiagov8 — CHANGES_REQUESTED** [review]
Hey @camillars 

1. Can you make the headings Normal capitalization instead of ALL UPPERCASE for consistency?
2. Can you fix the "Figure: Bad example - xxx" formatting too ;)

Thanks

File: rules/re-purpose-your-pillar-content-for-social-media/rule.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+::: good
+Figure: Good example, create a version of the video for Facebook, Twitter, LinkedIn, and Instagram
+
+:::
+
+3. Create a [Giphy](https://giphy.com/) with one of the main points and a moving background video from the pillar content. Write the copy for the post.
+4. Create a blog post from the audio
+
+* Use tools like [Rev](https://www.googleadservices.com/pagead/aclk?sa=L&ai=DChcSEwiumsqykr35AhUExpEKHU3jBy4YABADGgJjZQ&ohost=www.google.com&cid=CAESbeD2Ic_MW61jiFcDe5TxpDBil9rCr04QgZg-1IBYknvw25WRMTkVXqQCLNyhVj08eubxdsLPt-KmrUFF0xzX9q-H2Cw_v1m-07q6gs46AeYZUJ9pJfesiYhg8uiLl038VDm1PM7P5EOH2TVPBdc&sig=AOD64_0sUvgCggAIB04Iw_Pyh9lfx588YQ&q&adurl&ved=2ahUKEwiltcKykr35AhWyEbkGHb_TDpMQ0Qx6BAgEEAE) or [Descript](https://www.descript.com/) to transcribe the video file and re-write the transcript as a blog post
```
Comment (tiagov8) [inline rules/re-purpose-your-pillar-content-for-social-media/rule.md:60]: Fix link to be shorter: https://www.rev.com/services/video-transcription

File: rules/re-purpose-your-pillar-content-for-social-media/rule.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+archivedreason: null
+guid: f92caee6-b575-43bd-b3bd-a5ef44106f78
+---
+You spend a lot of time and money creating great video content, so you need to make sure that you get the most out of your time and money. 
+
+Content creation wizard Gary V (Vaynerchuk) has shared a great model for this on his blog called the 'reverse pyramid' where you start with a video, that you are able to repurpose into dozens of smaller pieces of content, contextual to the platforms that you distribute them to. 
+
+This could take many different forms — for example, if you’re not comfortable on video, you could record a podcast. You might even film yourself recording the podcast so you could have a video out of it as well. And from that video or audio clip, you can create content for Instagram, Twitter, Facebook, LinkedIn, and more. 
+
+You can find an explanation of the [Gary V Content Model here.](<* https://www.garyvaynerchuk.com/how-to-create-64-pieces-of-content-in-a-day>)
```
Comment (tiagov8) [inline rules/re-purpose-your-pillar-content-for-social-media/rule.md:29]: Fix markdown for link (ping me on Teams if in doubt)

File: rules/re-purpose-your-pillar-content-for-social-media/rule.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+* Daily vlog
+* Demo
+
+`youtube: https://www.youtube.com/embed/1fpIynISxnM`
+
+**Video: An example of a smaller video you can be used to drive traffic to your YouTube Channel**
+
+2. Create a quotable image with a nice background:
+
+* Create a version of the video for Facebook, Twitter, LinkedIn and Instagram (or the platforms you use). Write the copy for the post. ![](ASPNETCORE30_ENDPOINT_AUTHORIZATION_WITCHCRAFT.png)
```
Comment (tiagov8) [inline rules/re-purpose-your-pillar-content-for-social-media/rule.md:50]: The image markdown should be:
```
::: good
![Caption](path.png)
:::
```

Please fix

**Comment by camillars** [comment]
> Hey @camillars
> 
> 1. Can you make the headings Normal capitalization instead of ALL UPPERCASE for consistency?
> 2. Can you fix the "Figure: Bad example - xxx" formatting too ;)
> 
> Thanks

all done :)

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**